packages/wdio-spec-reporter/src/index.js

```diff
@@ -44,11 +45,15 @@ class SpecReporter extends WDIOReporter {
     }
 
     onTestPass () {
+        this.stateCounts.prevPassed = this.stateCounts.passed
```

Comment:
What if we switch line 48 and 49 so we can write line 54 as follows:

```js
if (this.stateCounts.prevPassed == this.stateCounts.passed) {
```

Reply:
I don't think that would work. ... For example, from test start (prevPassed, passed, failed all = 0), if the test failed immediately, then prevPassed === passed, and it would try to decrease passed to -1! Doing it as above means it will only decrease on failure after pass has erroneously increased.... prevPassed is a "holder" value for what passed should be, before it gets changed in error.